stl/inc/span

```diff
@@ -492,7 +527,7 @@ public:
             _STL_VERIFY(_Count <= this->size(), "Count out of range in span::first()");
         }
 #endif // _CONTAINER_DEBUG_LEVEL > 0
-        return {_Mydata, _Count};
+        return span<element_type, _Count>{_Mydata, _Count};

```

Comment:
I'm always tempted to use a deduced return type to avoid repetition in functions that return via an explicit initialization like this. Does anyone else agree/disagree?

Reply:
The reason I did not do this is that we would have two nearly identical versions. One with implicit construction and explicit type and one with explicitconstruction and implicit type.

I dont care which one to pick but I would rather go with a consisten one

Reply:
I went with deduced return type and explicit construction everywhere.